### PR TITLE
Use /bin/bash to execute the python command

### PR DESCRIPTION
**Pull request by workingloong**
Sometimes, we need to redirect the pod logs to a file in the disk and we can use bash redirection
command like ">> log.txt 2>&1". So we need to use /bin/bash in the command of the pod container. Then the args of the container is ["-c", "{COMMAND}"]

### CODE REVIEW

File: elasticdl/python/master/master.py [hunk 1]
```diff
@@ -462,15 +475,15 @@ def _create_instance_manager(self, args):
                 self.task_d,
                 job_name=args.job_name,
                 image_name=args.worker_image,
-                worker_command=worker_command,
+                worker_command=pod_command,
```
Comment (terrytangyuan) [inline elasticdl/python/master/master.py]: pod_command -> container_command

File: elasticdl/python/master/master.py [hunk 2]
```diff
@@ -449,6 +453,15 @@ def _create_instance_manager(self, args):
                     "--num_minibatches_per_task",
                     str(args.num_minibatches_per_task),
                 ]
+                ps_args = wrap_python_args_with_string(ps_args)
+                ps_args.insert(0, ps_client_command)
+
+            if args.log_file_path:
+                worker_args.append(">> {} 2>&1".format(args.log_file_path))
+                ps_args.append(">> {} 2>&1".format(args.log_file_path))
```
Comment (terrytangyuan) [inline elasticdl/python/master/master.py]: Refactor string to const

File: elasticdl/python/elasticdl/api.py [hunk 3]
```diff
@@ -182,6 +179,16 @@ def _submit_job(image_name, client_args, container_args):
         force_use_kube_config_file=client_args.force_use_kube_config_file,
     )
 
+    container_args = wrap_python_args_with_string(container_args)
+
+    master_client_command = "python -m elasticdl.python.master.main"
```
Comment (terrytangyuan) [inline elasticdl/python/elasticdl/api.py:184]: This appears in multiple places as well

File: elasticdl/python/common/args.py [hunk 4]
```diff
@@ -734,3 +737,38 @@ def _str_ignore_none(s):
         "--" + k if i % 2 == 0 else k for i, k in enumerate(arguments)
     ]
     return arguments
+
+
+def wrap_python_args_with_string(args):
```
Comment (terrytangyuan) [inline elasticdl/python/common/args.py:745]: Better to add simple test cases

File: elasticdl/python/common/args.py [hunk 5]
```diff
@@ -289,6 +289,9 @@ def add_common_params(parser):
         'by semi-colon used to debug , e.g. "param1=1; param2=2" '
         "Supported auxiliary parameters: disable_relaunch",
     )
+    parser.add_argument(
+        "--log_file_path", type=str, default="", help="The path to save logs",
```
Comment (terrytangyuan) [inline elasticdl/python/common/args.py]: What type of logs? e.g. stdout, stderr

File: elasticdl/python/elasticdl/api.py (hunk 3 above)
Comment (workingloong) [reply elasticdl/python/elasticdl/api.py:184]: It only appears here. Others are `python -m elasticdl.python.ps.main` and `python -m elasticdl.python.worker.main` which are responsible to start the ps process and the worker process.

Comment (terrytangyuan) [reply elasticdl/python/elasticdl/api.py:184]: You are correct. I was looking at the old code in the diff where this appears in multiple places.

File: elasticdl/python/common/args.py (hunk 4 above)
Comment (workingloong) [reply elasticdl/python/common/args.py:745]: Yeah!

File: elasticdl/python/common/args.py (hunk 5 above)
Comment (workingloong) [reply elasticdl/python/common/args.py]: The stdout and stderr logs.  I have added the comment in the help.

**Review by terrytangyuan — APPROVED** [review]

**Pull request merged**